dask_jobqueue/tests/test_sge.py

```diff
@@ -1,9 +1,70 @@
+import os
+from time import time, sleep
 
 import pytest
 
+from dask.distributed import Client
+from distributed.utils_test import loop  # noqa: F401
+
+from dask_jobqueue.sge import SGECluster
+
 pytestmark = pytest.mark.env("sge")
 
 
-def test_sge_placeholder():
-    # to test that CI is working
-    pass
+def test_basic(loop):  # noqa: F811
+    with SGECluster(walltime='00:02:00', threads=2, memory='7GB',
+                    loop=loop) as cluster:
+        with Client(cluster) as client:
+            workers = cluster.start_workers(2)
+            future = client.submit(lambda x: x + 1, 10)
+            assert future.result(60) == 11
+            assert cluster.jobs
+
+            info = client.scheduler_info()
+            w = list(info['workers'].values())[0]
+            assert w['memory_limit'] == 7e9
+            assert w['ncores'] == 2
+
+            cluster.stop_workers(workers)
+
+            start = time()
+            while len(client.scheduler_info()['workers']) > 0:
+                sleep(0.100)
+                assert time() < start + 10
+
+            assert not cluster.jobs
+
+
+def test_adaptive(loop):  # noqa: F811
+    with SGECluster(walltime='00:02:00', loop=loop) as cluster:
+        cluster.adapt()
+        with Client(cluster) as client:
+            future = client.submit(lambda x: x + 1, 10)
+            assert future.result(60) == 11
+
+            assert cluster.jobs
+
+            start = time()
+            while len(client.scheduler_info()['workers']) != cluster.config['processes']:
+                sleep(0.1)
+                assert time() < start + 10
+
+            del future
+
+            start = time()
+            while len(client.scheduler_info()['workers']) > 0:
+                sleep(0.100)
+                assert time() < start + 10
+
+            start = time()
+            while cluster.jobs:
+                sleep(0.100)
+                assert time() < start + 10
+
+
+@pytest.mark.skipif('SGE_ACCOUNT' in os.environ, reason='SGE_ACCOUNT defined')  # noqa: F811
+def test_errors(loop):
```

Comment:
I don't think this test is relevant anymore.

Reply:
Yeah I was wondering about that and I felt the same way.

Reply:
I think that this test came from a cluster where `project=` was necessary to run, and was often set implicitly with an environment variable (perhaps `PBS_ACCOUNT` on the cluster on which I was working at the time).  If this is not generally true for SGE clusters then I agree that this test should be removed.